fastFM/cpp_ffm.pxd

```diff
@@ -0,0 +1,30 @@
+# Author: Immanuel Bayer
+# License: BSD 3 clause
+#distutils: language=c++ 
+
+from libcpp.string cimport string
+
+cdef extern from "../../fastFM2/fastFM/fastfm.h" namespace "fastfm":
+
+    cdef cppclass Settings:
+        Settings() except +
+
+    cdef cppclass Data:
+        Data() except +
+        void add_design_matrix(const int rows, int cols, int nnz,
+                               int* outer_ptr, int* inter_ptr, double* data,
+                               int split)
```

Comment:
Please consider using the latest version of the api branch, I have done a few minor changes (for example dropping the split argument).